test/__snapshots__/autocomplete.test.ts.snap

```diff
@@ -20,7 +20,7 @@ exports[`autocomplete > should accept variants 1`] = `
 
 exports[`autocomplete > should provide autocomplete 1`] = `
 {
-  "align-": "align-base align-baseline align-bottom align-btm align-end align-inherit align-initial align-mid align-middle align-revert",
+  "align-": "align-10% align-100% align-20% align-30% align-40% align-50% align-60% align-70% align-80% align-90%",
```

Comment:
I am a bit unsure about this. Maybe we need a priority system, as I think the percentage usage should have a lower priority than the semantic ones.

Reply:
This is exactly what I am worried about, please allow me to deal with its priority in another PR.